### PR TITLE
feat: Setting of Routes Skeletons (REST API)

### PR DESCRIPTION
**Pull request by KushChauhanSS**
Created REST APIs.

https://redmine.successive.tech/issues/39523

### CODE REVIEW

File: src/controllers/trainee/routes.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Router } from 'express';
+import Trainee from './Controller';
```
Comment (adilsuccessive) [inline src/controllers/trainee/routes.ts]: import this as trainee (class instance).

File: src/router.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
+import { myRouter } from './controllers/trainee/index';
+
+const router = Router();
```
Comment (adilsuccessive) [inline src/router.ts]: type of router missing.

File: src/controllers/trainee/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export { default as myRouter } from './routes';
```
Comment (adilsuccessive) [inline src/controllers/trainee/index.ts]: export as traineeRouter for more readability.

**Review by adilsuccessive — CHANGES_REQUESTED** [review]

File: src/controllers/trainee/routes.ts (hunk 1 above)
Comment (KushChauhanSS) [reply src/controllers/trainee/routes.ts]: Done required changes.

File: src/router.ts (hunk 2 above)
Comment (KushChauhanSS) [reply src/router.ts]: Done required changes.

File: src/controllers/trainee/index.ts (hunk 3 above)
Comment (KushChauhanSS) [reply src/controllers/trainee/index.ts]: Done required changes.

**Review by adilsuccessive — APPROVED** [review]

**Pull request merged**